pkg/opte.template.p5m

```diff
@@ -33,5 +37,12 @@ file path=usr/lib/devfsadm/linkmod/SUNW_xde_link.so owner=root group=sys mode=07
 #:
```

Comment:
(darn github too short context)

curious if we still need the bypass stuff mention in the XXX above

Reply:
Yeah, we still do:

```
+ pkgdepend resolve -d packages -s resolve.p5m opte.generate.p5m
/develop/gits/opte/pkg/opte.generate.p5m has unresolved dependency '
    depend type=require fmri=__TBD pkg.debug.depend.file=dld \
        pkg.debug.depend.reason=kernel/drv/amd64/xde \
        pkg.debug.depend.type=elf \
        pkg.debug.depend.path=kernel/misc/amd64 \
        pkg.debug.depend.path=usr/kernel/misc/amd64
' under the following combinations of variants:
variant.debug.illumos:false variant.opensolaris.zone:global
/develop/gits/opte/pkg/opte.generate.p5m has unresolved dependency '
    depend type=require fmri=__TBD pkg.debug.depend.file=dld \
        pkg.debug.depend.reason=kernel/drv/amd64/xde \
        pkg.debug.depend.type=elf \
        pkg.debug.depend.path=kernel/misc/amd64 \
        pkg.debug.depend.path=usr/kernel/misc/amd64
' under the following combinations of variants:
variant.debug.illumos:true variant.opensolaris.zone:global
/develop/gits/opte/pkg/opte.generate.p5m has unresolved dependency '
    depend type=require fmri=__TBD pkg.debug.depend.file=mac \
        pkg.debug.depend.reason=kernel/drv/amd64/xde \
        pkg.debug.depend.type=elf \
        pkg.debug.depend.path=kernel/drv/amd64 \
        pkg.debug.depend.path=usr/kernel/drv/amd64
' under the following combinations of variants:
variant.debug.illumos:false variant.opensolaris.zone:global
/develop/gits/opte/pkg/opte.generate.p5m has unresolved dependency '
    depend type=require fmri=__TBD pkg.debug.depend.file=mac \
        pkg.debug.depend.reason=kernel/drv/amd64/xde \
        pkg.debug.depend.type=elf \
        pkg.debug.depend.path=kernel/drv/amd64 \
        pkg.debug.depend.path=usr/kernel/drv/amd64
' under the following combinations of variants:
variant.debug.illumos:true variant.opensolaris.zone:global
```

I briefly tested different `pkgdepend -k <path>` and `pkg.depend.runpath=<path>:$PKGDEPEND_RUNPATH` to no luck.

Reply:
ah, unfortunate :/ i don't know enough about illumos packaging, so perhaps worth a question in chat. in any case, doesn't need to block this PR

Reply:
@citrus-it can likely help explain what's required and what's going on and may have some ideas here.